SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/MinProximityIntersection.h

```diff
@@ -27,6 +27,29 @@
 namespace sofa::component::collision
 {
 
+/**
+ * Basic intersection methods using proximities
+ * It uses proximities on cubes and spheres, but supported pairs of collision models can
+ * be extended. For example, see MeshMinProximityIntersection which adds support for
+ * additional types of intersection.
+ *
+ * Supported by default:
+ * - Cube/Cube
+ * - Sphere/Sphere (rigid or vec3)
+ * MeshMinProximityIntersection adds support for:
+ * - Point/Point (if usePointPoint is true)
+ * - Line/Point (if useLinePoint is true)
+ * - Line/Line (if useLineLine is true)
+ * - Triangle/Point
+ * - Sphere/Point (if useSphereTriangle is true)
+ * - RigidSphere/Point (if useSphereTriangle is true)
+ * - Triangle/Sphere (if useSphereTriangle is true)
+ * - Triangle/RigidSphere (if useSphereTriangle is true)
+ * - Line/Sphere (if useSphereTriangle is true)
+ * - Line/RigidSphere (if useSphereTriangle is true)
+ * Note that MeshMinProximityIntersection ignores Triangle/Line and Triangle/Triangle intersections.
```

Comment:
just for my knowledge, do you know why it is ignored?
The comment is very useful, thanks!

Reply:
No I don't, but I know that if you want Triangle/Triangle intersection, you can go to `NewProximityIntersection`.